source/products/hansemerkur/cat/rest-api.rst

```diff
@@ -0,0 +1,210 @@
+      -H 'Content-Type: application/json' \
+      -u <SECRET_KEY>: \
+      -d '{
+        "item_id": "<ITEM_ID>",
+        "touchpoint_id": "<TOUCHPOINT_ID>",
+        "subscription_plan_id": "<SUBSCRIPTION_ID>",
+        "data": {
+            "coverage_type": "best",
+            "cat_breed": "dai_some_id_32_chars_long_______",
+            "cat_dob": "2019-01-01",
+            "dental_surgery": true,
+            "payment_frequency": "yearly",
+            "policy_start_date": "2020-09-27",
+            "deductibles": 25000,
+            "cat_type": "home"
+        }
+    }'
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code:: javascript
+
+    {
+        "token": "<TOKEN>",
+        "gross_payment_amount": 11255,
+        "extra_data": {
+            "gross_premium": 11255,
+            "premium_tax": 1797,
+            "net_premium": 9458,
+            "tax_rate": 0.19,
+            "gross_premium_cat": 11255,
+            "gross_premium_dental_surgery": 2856,
+            "cat_age": 1,
+            "gross_premium_dental_surgery_true": 2856,
+            "gross_premium_smart": 5356,
+            "gross_premium_easy": 7766,
+            "gross_premium_best": 8399
+        }
+    }
+
+Create Unpaid Policy Request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+JSON data posted to /policies on creation of unpaid policy.
+
+.. csv-table::
+   :header: "Name", "Type", "Description", "Example Value"
+   :widths: 20, 20, 80, 20
+
+   "first_name", "string", "First name", "John"
+   "last_name", "string", "Last name", "Tester"
+   "email", "string", "Email", "test@kasko.io"
+   "cat_name", "string", "Dog name", "Rex"
+   "cat_gender", "string", "male or female", "male"
+   "cat_with_chip", "string", "Chip type - tattoo,characteristics,chip", "tattoo"
+   "cat_chip_number", "string", "Chip number", "123456789123456"
+   "cat_tattoo_number", "string", "Tatoo number", "ABC123"
+   "fur_color", "string", "Fur Color", "brown"
+   "template", "string", "Template", "template"
+   "special_characteristics", "string", "Special cat characteristics", "Special thing"
+   "cat_health", "boolean", "", "true"
+   "previous_insurance", "boolean", "", "false"
+   "previous_insurance_name", "string", "", ""
+   "previous_insurance_ended_by", "string", "", ""
+   "salutation", "string", "mr / ms", "mr"
+   "dob", "string", "Date of birth of dog", "2017-11-06"
+   "house_number", "string", "House number", "ABC"
+   "street", "string", "Street number", "DEF"
+   "city", "string", "City name", "London"
+   "postcode", "string", "Postal code", "12345"
+   "phone", "string", "Phone number", "+999 233445566"
+   "consultation", "boolean", "Is consultation needed", "false"
+   "coverage_to_1000", "boolean", "", "true"
+   "coverage_to_2000", "boolean", "", "true"
+   "coverage_to_5000", "boolean", "", "true"
+   "adnr_number", "string", "", "12"
+
+Example Request
+~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+    curl -X POST \
+            'https://api.kasko.io/policies' \
```

Comment:
Is this the right api url? Below in example response "links" use different api url.

Reply:
Yeah, they both work!